pkgs/applications/blockchains/chia/default.nix

```
@@ -0,0 +1,68 @@
+{ lib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "chia";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "Chia-Network";
+    repo = "chia-blockchain";
+    rev = version;
+    sha256 = "ZUxWOlJGQpeQCtWt0PSdcbMackHdeuNFkxHvYDPcU8Y=";
+  };
+
+  patches = [
+    ./dependencies.patch
+  ];
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools_scm
+  ];
+
+  # give a hint to setuptools_scm on package version
+  preBuild = ''
+    export SETUPTOOLS_SCM_PRETEND_VERSION="v${version}"
+  '';
```

I think this looks cleaner
```suggestion
  SETUPTOOLS_SCM_PRETEND_VERSION = "v${version}";
```